imports/server/vehicle.lua

```diff
@@ -32,8 +32,24 @@ function OxVehicle:getState()
     return Entity(self.entity).state
 end
 
-function OxVehicle:getCoords()
-    return GetEntityCoords(self.entity)
+function OxVehicle:getCoords(heading)
+    local coords = GetEntityCoords(self.entity)
+    if heading == true then
```

Comment:
```suggestion
    if heading then
```
Same shit as the other one